share/spice/news/news.js

```diff
@@ -6,61 +6,66 @@
             return Spice.failed('news');
         }
 
-        // Words that we have to skip in DDG.isRelevant.
-        var skip = [
-            "news",
-            "headline",
-            "headlines",
-            "latest",
-            "breaking",
-            "update",
-            "s:d",
-            "sort:date"
-        ];
+        var useRelevancy = DDG.opensearch.installed.experiment === "static_news" && DDG.opensearch.installed.variant === 'b',
```

Comment:
Wouldn't this switch off this logic for everybody but the `b` variant? Is that intended?